how-to/workspace-platform-starter/public/manifest.fin.json

```diff
@@ -1551,6 +1551,17 @@
 					}
 				}
 			]
+		},
+		"snapProvider": {
+			"enabled": true,
```

Comment:
Should this be false if the url for the asset isn't provided?

Reply:
Done